tools/hack/deployment-exists.sh

```diff
@@ -3,9 +3,8 @@
 DEPLOYMENT_LABEL_SELECTOR=$1
 DEPLOYMENT_NAMESPACE=$2
 
-
 # Timeout for deployment to exist (in seconds)
-exist_timeout=25
+exist_timeout=300
```

Comment:
configmap updates to the pod take at least 30 seconds to effect in pod, increase timeout for this.